galsim/bandpass.py

```diff
@@ -413,14 +413,37 @@ def truncate(self, blue_limit=None, red_limit=None, relative_throughput=None):
         This function does not remove any intermediate wavelength ranges, but see thin() for
         a method that can thin out the intermediate values.
 
-        @param blue_limit       Truncate blue side of bandpass here. [default: None]
-        @param red_limit        Truncate red side of bandpass here. [default: None]
+        When truncating a bandpass that already has an assigned zeropoint, there are several
+        possibilities for what should happen to the new (returned) bandpass by default.  If red
+        and/or blue limits are given, then the new bandpass will have no assigned zeropoint because
+        it is difficult to predict what should happen if the bandpass is being arbitrarily
+        truncated.  If `relative_throughput` is given, often corresponding to low-level truncation
+        that results in little change in observed quantities, then the new bandpass is assigned the
+        same zeropoint as the original.  This default behavior is called 'auto'.  The user can also
+        give boolean True or False values.
+
+        @param blue_limit           Truncate blue side of bandpass here. [default: None]
```

Comment:
I'm sure this is my previous oversight, but I think it would be good to indicate the units for blue_limit and red_limit here, which I'm pretty sure are always nanometers (regardless of the wave_type of the bandpass).